_studio/mfx_lib/CMakeLists.txt

```diff
@@ -71,7 +71,7 @@ append_property( mfx_common_hw COMPILE_FLAGS " -DMFX_RT" )
 
 ### libmfxhw
 list( APPEND mdirs
-  scheduler vpp decode/vc1
+  scheduler/linux vpp decode/vc1
```

Comment:
Build for the "move" patch failed. We need to have patches buildable independently. Please, make a year copyright change here in this commit.

Reply:
Changed copyright years for moved files